tachyon/math/elliptic_curves/short_weierstrass/affine_point.h

```diff
@@ -104,6 +106,21 @@ class AffinePoint<
                        point.y_);
   }
 
+  template <typename ScalarFieldContainer, typename AffineContainer>
+  [[nodiscard]] constexpr static bool BatchMapScalarFieldToPoint(
```

Comment:
I'm not sure why this function is named `BatchMapScalarFieldToPoint`. Can you elaborate on it a bit more please?

Reply:
This name is derived from the fact that map `scalar field` of elliptic curve to `point`.

s₀, s₁, ..., sₙ₋₁ -> s₀G, s₁G, ..., sₙ₋₁G